content/en/docs/topics/kubernetes_apis.md

```diff
@@ -0,0 +1,141 @@
+deprecation policy and process to inform users of how changes to APIs are
+implemented. In other words, consumers of your API need to know in advance and
+in what release an API will be removed or changed. This removes the element of
+surprise and breaking changes to consumers.
+
+The [Kubernetes deprecation policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/)
+documents how Kubernetes handles the changes to its API versions. The policy
+for deprecation states the timeframe that API versions will be supported following
+a deprecation announcement. It is therefore important to be aware of deprecation
+announcements and know when API versions will be removed, to help minimize the
+effect.
+
+This is an example of an announcement [for the removal of deprecated API versions in Kubernetes 1.16](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/)
+and was advertised a few months prior to the release. These API versions would
+have been announced for deprecation prior to this again. This shows that there
+is a good policy in place which informs consumers of API version support. 
+
+Helm templates specify a [Kubernetes API group](https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-groups)
+when defining a Kubernetes object, similar to a Kubernetes manifest file. It is
+specified in the `apiVersion` field of the template and it identifies the API
+version of the Kubernetes object. This means that Helm users and chart maintainers
+need to be aware when Kubernetes API versions have been deprecated and in what
+Kubernetes version they will be removed.
+
+## Chart Maintainers
+
+You should audit your charts checking for Kubernetes API versions that are
+deprecated or are removed in a Kubernetes version. The API versions found as
+due to be or that are now out of support, should be updated to the supported
+version and a new version of the chart released. The API version is defined by
+the `kind` and `apiVersion` fields. For example, here is a removed `Deployment`
+object API version in Kubernetes 1.16:
+
+```yaml
+apiVersion: apps/v1beta1
+kind: Deployment
+```
+
+## Helm Users
+
+You should audit the charts that you use (similar to the [chart maintainer](#chart-maintainers))
+and identify any charts where API versions are deprecated or removed in a
+Kubernetes version. For the charts identified, you need to check for the latest
+version of the chart (which has supported API versions) or update the chart
+yourself.
+
+Additionally, you also need to audit any charts deployed (i.e. Helm releases)
+checking again for any deprecated or removed API versions. This can be done by
+getting details of a release using the `helm get manifest` command.
+
+The means for updating a Helm release to supported APIs depends on your findings
+as follows:
+
+1. If you find deprecated API versions only then:
+  - Perform a `helm upgrade` with a version of the chart with supported Kubernetes
```

Comment:
does this work now?

Reply:
Yes as the APIs are deprecated and not yet removed from a Kubernetes version. This means then that you are still use the deprecated API version and update it to the new supported version which replaces it.

Reply:
To clarify that this would always have worked.